src/main/java/com/synopsys/integration/jira/common/cloud/service/JiraCloudServiceFactory.java

```diff
@@ -82,8 +83,8 @@ public ProjectService createProjectService() {
         return new ProjectService(createJiraService());
     }
 
-    public JiraAppService createJiraAppService() {
-        return new JiraAppService(gson, httpClient, createJiraService());
+    public PluginManagerService createPluginManagerService() {
```

Comment:
For the shared services, can we create a common ServiceFactory that we pull these classes from? I see a number of imports for Services that aren't necessarily specific to Jira Cloud.